arch/arm/src/rp23xx/rp23xx_start.c

```diff
@@ -86,8 +87,21 @@ void __start(void)
 #ifdef CONFIG_BOOT_RUNFROMFLASH
   const uint32_t *src;
 #endif
-  uint32_t *dest;
-  int i;
+  uint32_t            *dest;
```

Comment:
let's align with line 88
```suggestion
  uint32_t       *dest;
```